code/modules/mob/living/carbon/human/species_types/zombies.dm

```diff
@@ -66,8 +66,9 @@
 		var/heal_amt = heal_rate
 		if(C.InCritical())
 			heal_amt *= 2
-		C.heal_overall_damage(heal_amt,heal_amt)
+		C.heal_overall_damage(heal_amt,heal_amt,heal_amt)
```

Comment:
Intended? If so update changelog and such please

Reply:
ah, no. That was a "working on PR at 2AM, what was this third arg for again" but I forgot to delete it.

Reply:
![image](https://github.com/BeeStation/BeeStation-Hornet/assets/9547572/e90e2819-021d-4228-86ff-3a468ce4392d)

I had done this to see, then had compile error from comma and "fixed" it. Reverted with latest push now